code/datums/shuttles.dm

```diff
@@ -1,6 +1,8 @@
 /datum/map_template/shuttle
 	name = "Base Shuttle Template"
 	var/prefix = "_maps/shuttles/"
+	///Name of the file to load (without the .dmm extension)
+	var/file_name
```

Comment:
If I remember right, the file name was chosen off the ID. Since there can be multiple Alamos in different configurations, this was an issue. Tadpole did it by just appending a suffix but seems stinky when you can just declare the actual file name, the suffix isn't needed for anything else.